.env.example

```diff
@@ -26,6 +26,8 @@ MAIL_GOOGLE_USER='key_MAIL_GOOGLE_USER'
 MAIL_GOOGLE_PASS='key_MAIL_GOOGLE_PASS'
 MIN_LOG_DURATION='200'
 PORT='3000'
+POSTGRES_URL='postgresql://postgres:password@localhost:5432/web'
```

Comment:
let's have a single source of truth for the password. If it's already in the POSTGRES_URL, then no need for an additional env var

Reply:
@mattkrick great idea, however after looking into it it seems that the two references different env vars and so the value may not always coincide. See note 2 below. 

given this info, i'm comfortable with leaving the 2 as separate. what do you think?

from https://hub.docker.com/_/postgres
```
POSTGRES_PASSWORD
This environment variable is required for you to use the PostgreSQL image. It must not be empty or undefined. This environment variable sets the superuser password for PostgreSQL. The default superuser is defined by the POSTGRES_USER environment variable.

Note 1: The PostgreSQL image sets up trust authentication locally so you may notice a password is not required when connecting from localhost (inside the same container). However, a password will be required if connecting from a different host/container.

Note 2: This variable defines the superuser password in the PostgreSQL instance, as set by the initdb script during initial container startup. It has no effect on the PGPASSWORD environment variable that may be used by the psql client at runtime, as described at https://www.postgresql.org/docs/current/libpq-envars.html. PGPASSWORD, if used, will be specified as a separate environment variable.
```

Reply:
ah, that's an interesting thing they're doing with the docker image.
so the info we need is:
- SU user
- SU password
- user
- password
- host
- database
- port

from the docs, it looks like if we don't provide a `POSTGRES_USER` or `POSTGRES_DB`, then it'll create a database named `postgres`. 
does that mean if we include something like `POSTGRES_DB='web'` then we can get rid of the init.sql?  that may be cleaner

Reply:
> to DRY things out a bit, perhaps we just create these env vars instead of the URL? i don't have any tension sharing the SU user/pass for the app since the app will need SU permissions anyways.

@mattkrick by DRY, do you mean specifying the env vars `PGUSER`, `PGHOST`, `PGDATABASE`, `PGPORT` separately save `PGPASSWORD` and then when instantiating the pg client, programmatically specifying that `PGPASSWORD` equals `POSTGRES_PASSWORD`?

Reply:
the docker image will need POSTGRES_USER and POSTGRE_DB, too. we can just pass those in as params at runtime

Reply:
just wanted to clarify: by `pass those in as params at runtime`, do you mean setting the docker postgres image env vars not in `.env` file but either in the Dockerfile, docker-compose file, or injecting them by always calling the docker-compose command with exported env vars?

wanted to make sure bc the comment seems to contradict an earlier suggestion to take the env vars out of Dockerfile and only set them in `.env` file.

also: not sure abt you but i'm finding it challenging to talk abt dry setup config stuff in comments, lmk if you prefer to take this sync.